tests/PageApprovalsIntegrationTest.php

```diff
@@ -4,6 +4,15 @@
 
 namespace ProfessionalWiki\PageApprovals\Tests;
 
+use MediaWiki\MediaWikiServices;
+use Title;
+
 class PageApprovalsIntegrationTest extends \MediaWikiIntegrationTestCase {
 
+	protected function getIdOfExistingPage( string $titleText ): int {
```

Comment:
This can be read as the page needing to exist already.

createPageAndReturnId or simply createPage could do the trick. The latter might conflict with other methods tho.